pkg/component/autoscaling/vpa/vpa_test.go

```diff
@@ -1680,6 +1742,89 @@ var _ = Describe("VPA", func() {
 				managedResourceName = "shoot-core-vpa"
 			})
 
+			Context("Different ServiceMonitor labels", func() {
```

Comment:
```suggestion
			Context("when deploying ServiceMonitors", func() {
```

Reply:
Thanks for the suggested wording. You can find the fixed version in my latest updates.